site/components/pages/Retirements/SingleRetirement/index.tsx

```diff
@@ -126,79 +137,96 @@ export const SingleRetirementPage: NextPage<SingleRetirementPageProps> = (
           }
         />
         <div className={styles.retirementContent}>
-          <RetirementMessage
-            message={
-              retireData.retirementMessage ||
-              t({
-                id: "retirement.single.retirementMessage.placeholder",
-                message: "No retirement message provided",
-              })
-            }
-          />
-          <RetirementValue
-            value={retireData.amount}
-            label={retireData.tokenLabel}
-            icon={retireData.tokenIcon}
-          />
-          <div className={styles.metaData}>
-            <div className="column">
-              <TextGroup
-                title={
-                  <Trans id="retirement.single.beneficiary.title">
-                    Beneficiary
-                  </Trans>
-                }
-                text={
-                  retireData.beneficiaryName ||
-                  t({
-                    id: "retirement.single.beneficiary.placeholder",
-                    message: "No beneficiary name provided",
-                  })
-                }
-              />
-              <TextGroup
-                title={
-                  <Trans id="retirement.single.beneficiaryAddress.title">
-                    Beneficiary Address
-                  </Trans>
-                }
-                text={
-                  <Link
-                    href={`/retirements/${
-                      nameserviceDomain || beneficiaryAddress
-                    }`}
-                    className="address"
-                  >
-                    {nameserviceDomain || concatAddress(beneficiaryAddress)}
-                  </Link>
-                }
-              />
+          <RetirementMessage message={retirement.retirementMessage} />
```

Comment:
Message can be empty.
You removed the fallback "No retirement message provided"

See result [here](https://klimadao-site-git-atmos-page-performance-klimadao.vercel.app/en/retirements/0x2d2f9d78a0ed19bceae13bb81daee2825a1c53fc/4)

<img width="359" alt="Bildschirm­foto 2023-02-20 um 13 23 18" src="https://user-images.githubusercontent.com/95881624/220106565-7149657e-552c-4393-864a-d51aeaf228f4.png">